generate-third-party-licenses.cjs

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+
+const exec = require("util").promisify(require("child_process").exec);
+const fs = require("fs");
+
+function processDeps(deps) {
+  deps = deps
+    .filter((dep) => dep.path !== undefined)
+    .map(({ name, path }) => ({
+      name,
+      licenseFiles: fs
+        .readdirSync(path)
+        .map((file) => `${path}/${file}`)
+        .filter((file) =>
+          /^(licen[sc]e|copying)/i.test(
+            file.substring(file.lastIndexOf("/") + 1)
+          )
+        ),
+    }));
+
+  let thirdPartyLicenses = "";
```

Comment:
There are a couple repeated Node dependencies and as a result duplicated licenses, maybe we should deduplicate them somewhere here?

Reply:
If it's not too much trouble, but certainly not a high prio thing.

Reply:
I think I can figure it out. Also, displaying the versions used would be great.